product_docs/docs/eprs/6.2/07_common_operations/08_replicating_ddl_changes/01_ddl_change_replication.mdx

```diff
@@ -22,4 +22,6 @@ The following is the series of steps that occur during the DDL change replicatio
 -   The in-memory table metadata definition is refreshed to reflect the DDL change. The user is informed of successful completion of the operation.
 -   If an error occurs during the prior steps, any changes up to that point are rolled back so that the publication table, replication triggers, and shadow table are reverted back to their original state prior to the start of this operation. If one or more databases goes down before completion of the operation, the publication is marked as dirty to avoid further replication events.
 
-The following section describes how to initiate DDL change replication using the xDB Replication Console.
+!!! Note
+   When you execute the alterDDL command, non-MDN nodes usually don't have any CDC changes during the MMR setup. If there are any changes on the altered table in the non-MDN node, it can result in data loss and a break in replication. 
```

Comment:
Please update the note as below

When you execute the alterDDL command, non-MDN nodes should not have any CDC changes during that window for MMR setup. If there are any changes on the altered table during that window in the non-mdn nodes, it may result in data loss and a break in replication.

Reply:
To keep the note easier to read can we say:
- "during the MMR setup window"
- to begin the second sentence, "If there are any changes, it can result" 

I think these would make it easier to read and complies with our Style Guide. But if it changes the meaning, then we can work on it.

Reply:
- "during the MMR setup" does not convey the right meaning. 
- There is no guarantee that this will result in issue .There is a possibility of issue if changes are made on the altered table during the alterDDL operation. So we need to be specific about the time window of this operation .

Reply:
Thanks for the feedback